source/git/source_test.go

```diff
@@ -635,6 +714,7 @@ func setupGitRepo(t *testing.T) gitRepoFixture {
 		"git add -A",
 		"git commit -m withsub",
 		"git checkout master",
+		// "git log --oneline --graph --decorate=full --all",
```

Comment:
Is this a leftover debug print?

Reply:
This line can be used to update the comment showing the git structure in line 686 if updates are made